pkg/receiver/gitproviderreceiver/internal/scraper/gitlabscraper/genqlient.graphql

```diff
@@ -1,51 +1,57 @@
-query getGroupProjects($fullPath: ID!) {
+query getAllGroupProjects(
+  $fullPath: ID!,
+  # @genqlient(pointer: true)
+  $after: String
+) {
   group(fullPath: $fullPath) {
-    projects {
+    projects(includeSubgroups: true, after: $after) {
+      count
+
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+
       nodes {
         name
+        fullPath
         createdAt
         lastActivityAt
       }
     }
   }
 }
 
-query getProject($fullPath: ID!) {
+query getProjectsByTopic($org: String!, $topics: [String!]) {
```

Comment:
Is there a way to do this as a singular query string being passed in like we do with the github scraper?

Reply:
Let me see what I can do.

Reply:
![image](https://github.com/liatrio/liatrio-otel-collector/assets/6931731/ee9e7831-7f2a-47b3-b688-b1fc990b27d0)

Looks like if we want to actually pull projects by Topics, we have to do it this way.

That said, I don't see where we're currently using this query in the scraper so it may be a leftover.

Reply:
ok. it definitely would be ideal to be able to look at a group by default, but be able to add a search query (like by topic) in the event that you want to narrow down. 

For example, if we had a liatrio group on gitlab, but we had 500 repo,s we'd only want to look at ones for a specific team(topic).

Reply:
created new piece of work for this